Updated the information on the Organization Card.

AmitSharma512 opened the pull request:
**What kind of change does this PR introduce?**

Updated the information on the Organization Card.

**Issue Number:**

Fixes #1538 

**Snapshots/Videos:**

![Screenshot 2024-02-27 023910](https://github.com/PalisadoesFoundation/talawa-admin/assets/110725065/e269cd1b-a575-4038-b9a7-27ac346a6cfe)

**Does this PR introduce a breaking change?**

No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

AmitSharma512 commented:
@palisadoes I guess there is some issue with the PR workflow because the readme file is automatically changing while making a PR.

Cioppolo14 commented:
@beingnoble03 @noman2002 Can you review this PR?

AmitSharma512 commented:
@noman2002 @beingnoble03  I guess there is some issue with the PR workflow because the readme file automatically changes while making a PR, and some "--" signs are added to the readme file in the commits.

palisadoes commented:
1. The screen shot does not comply with what the issue requests.
2. Please update the PR with the necessary changes and/or the screenshot

AmitSharma512 commented:
@palisadoes this design is from the official figma file of Palisadoes, the design is same with the figma file.

palisadoes commented:
1. True, but the issue identifies deficiencies in the design.
2. Add a link to your Figma with the modifications so that we can update ours

noman2002 commented on README.md:
```diff
@@ -25,7 +25,7 @@ Core features include:
 
 - [Talawa Components](#talawa-components)
-- [Documentation](#documentation)
+- [Documentation -](#documentation--)
```
Undo this.

AmitSharma512 replied:
@noman2002 I have reverted the changes; please take a look at them.

palisadoes commented:
Please remove the eslint disable comment so that we can review and merge
```
talawa-admin/src/screens/OrganizationPeople/OrganizationPeople.tsx 
```

AmitSharma512 commented:
> Please remove the eslint disable comment so that we can review and merge
> 
> ```
> talawa-admin/src/screens/OrganizationPeople/OrganizationPeople.tsx 
> ```

Sir, which comments must be removed because I have not placed any comments in my PR?

palisadoes commented:
1. Check the failed tests
2. We just added an eslint detection rule. This PR was one of the first after it was implemented. It is a minor exception.
3. Please fix

AmitSharma512 commented:
> 1. Check the failed tests
> 2. We just added an eslint detection rule. This PR was one of the first after it was implemented. It is a minor exception.
> 3. Please fix

I have changed the condition for Camel Case rules in the eslintrc.json file because the Camel Case rule is causing some problems in the file.

palisadoes commented on .eslintrc.json:
```diff
@@ -91,7 +92,7 @@
       {
         "selector": "variable",
-        "format": ["camelCase", "UPPER_CASE"],
+        "format": ["camelCase", "UPPER_CASE", "PascalCase"],
```
1. Can you revert the eslint changes and fix the file?
2. You seem to have introduced multiple code style rules for the entire repository based on one file.
3. Please fix the offending file

AmitSharma512 replied:
@palisadoes I think that these changes are necessary because we need to make the first letter capital while creating a variable or component, and if we do so with the previous eslint rules we cannot create it because of the CamelCase Conditon, that's why previously the lines are commented in order to ignore the eslint conditons.
